packages/story-editor/src/components/canvas/useInsertElement.ts

```diff
@@ -76,9 +83,18 @@ function useInsertElement() {
       const elementId = insertAsBackground ? backgroundElementId : id;
 
       // Auto-play on insert.
-      if (type === 'video' && resource?.src && !resource.isPlaceholder) {
+      if (
+        elementId &&
+        elementIs.media(element) &&
+        elementIs.sequenceMedia(element) &&
+        type === 'video' &&
```

Comment:
Don't we have a better test for this? We should at least use ElementType enum no?